src/GitHub/Data/PullRequests.hs

```diff
@@ -180,8 +184,22 @@ instance Binary PullRequestReference
 -- JSON instances
 -------------------------------------------------------------------------------
 
+-- | Helper function, reads either the "assignee" OR "assigneed" OR
+-- both from a JSON object.
+getAssignees :: Object -> Parser (Vector SimpleUser)
```

Comment:
But it's current variant https://api.github.com/repos/octocat/Hello-World/pulls **does** have `assignees`. *GitHub documentation is outdated, because examples are manually written*.